terminal/command.go

```diff
@@ -503,6 +504,29 @@ func stackCommand(client service.Client, args ...string) error {
 	return nil
 }
 
+func listCommand(client service.Client, args ...string) error {
+	if len(args) == 0 {
+		state, err := client.GetState()
+		if err != nil {
+			return err
+		}
+		printcontext(state)
+		return nil
+	}
+
+	locs, err := client.FindLocation(args[0])
+	if err != nil {
+		return err
+	}
+
+	if len(locs) > 0 {
+		return fmt.Errorf("Ambiguous location: %s", args[0])
+	}
+
+	printfile(locs[0].File, locs[0].Line)
```

Comment:
It would be nice to pass a bool to printfile to tell it whether or not to print the current location arrow, since when we `list main.main` or whatever we really don't want the arrow printed.